tscreen.go

```diff
@@ -948,8 +948,9 @@ func (t *tScreen) enableMouse(f MouseFlags) {
 		if f&MouseMotionEvents != 0 {
 			t.TPuts("\x1b[?1003h")
 		}
-
-		t.TPuts("\x1b[?1006h")
+		if f != 0 {
```

Comment:
This is how we undo mouse hooks when returning back to a normal shell. 

Doing it this way is unacceptable.

Reply:
To be clear, see my other comments -- but if we want to "not" do this, it needs to be conditionalized -- either on the state of the mouse tracking at startup, or upon some other user-supplied preference.

Reply:
Sorry for my terrible english.

What I wanted to say is that mouse tracking should be undo when returning to a normal shell, but 1006 remains set.